compiler/src/dotty/tools/dotc/Compiler.scala

```diff
@@ -70,7 +70,6 @@ class Compiler {
            new CrossCastAnd,        // Normalize selections involving intersection types.
            new Splitter),           // Expand selections involving union types into conditionals
       List(new VCInlineMethods,     // Inlines calls to value class methods
-           new IsInstanceOfEvaluator, // Issues warnings when unreachable statements are present in match/if expressions
```

Comment:
Why is `IsInstanceOfEvaluator` removed? I see that there are some modifications to it.